hangman/hangman.py

```diff
@@ -21,9 +21,11 @@ def hangman():
     Processes the user input, with the target word
     to determine which letter is guessed correctly.
     """
+    times = 10
     word=""
     count = 0
     while True:
+        print(count)
```

Comment:
Either remove or print actual information that is useful to the user

Reply:
This was a mistake,
 Used it for testing.